public-user/molecule/default/molecule.yml

```diff
@@ -0,0 +1,30 @@
+---
+dependency:
+  name: galaxy
+driver:
+  name: docker
+lint:
+  name: yamllint
+platforms:
+  - name: omero-all
```

Comment:
In line with most molecule configs could you change this to something related to what's being tested? The old molecule file had `omero-server`, though `omero-public` might be better.